front/src/modules/people/components/__stories__/PhoneEditableField.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import { BrowserRouter } from 'react-router-dom';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { PhoneCellEdit } from '@/ui/table/editable-cell/type/components/PhoneCellEdit';
+import { ComponentDecorator } from '~/testing/decorators/ComponentDecorator';
+
+const meta: Meta<typeof PhoneCellEdit> = {
+  title: 'Modules/People/EditableFields/PhoneEditableField',
+  component: PhoneCellEdit,
+  decorators: [
+    (Story) => (
```

Comment:
why is Browser required to test this component, I feel ComponentDecorator is enough